spec/features/author_creation_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+feature 'Creating Authors' do
+  let(:user) { create :user }
+  let(:author) { attributes_for :author }
+
+  before do
+    sign_in user
+  end
+
+  scenario 'Visiting the author page' do
+    visit new_admin_author_path
+    expect(page).to have_link t('authors.pluralized_title')
+    fill_in 'author_name', with: author
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.